src/Drivers/SymfonyTinkerwellDriver.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use Tinkerwell\ContextMenu\Label;
+use Tinkerwell\ContextMenu\OpenURL;
+use Tinkerwell\ContextMenu\SetCode;
+use Tinkerwell\ContextMenu\Submenu;
+use Symfony\Component\HttpFoundation\Request;
+
+class SymfonyTinkerwellDriver extends TinkerwellDriver
+{
+    public function canBootstrap($projectPath)
+    {
+        return file_exists($projectPath . '/public/index.php');
```

Comment:
I think that this check is not specific enough to check that the framework is Symfony.
You can maybe check that $projectPath . '/symfony.lock' and $projectPath . '/config/bootstrap.php' exists.

Reply:
After checking with a Symfony user, testing `symfony.lock` or `bin/console` is a good start.
If the config/bootstrap.php file is required in the bootstrap method, it might be good to test it as well.

Reply:
Thanks, totally looked over these, added them!